readme.md

```diff
@@ -391,6 +391,7 @@ this:
 - For the widget in the list, there are additional `_meta` properties for that
 specific resource, as well as some actual data properties
 (`color` and `make`).
+    - The `_meta` section in each resource contains properties generated by the API service. They cannot be created or updated by the client. They are not to be confused with data properties of the same name.
```

Comment:
Thanks for adding this. I've been meaning to note something similar. I was going to approach it thinking about that section as computed properties:

"`_meta` properties can be thought of as "computed" properties about the resource: avoid using these properties for data that the client could not itself compute. They are best used if that computation would be expensive for the client but is cheap for the server."

If you can find a nice way to work this into your statement, that would be ideal I think. Yours also addresses some properties that don't fit this (UUID, timestamps, etc.) that I think are valid as well for the reasons you list (read-only, service-managed properties).